src/language/lang_tr.php

```diff
@@ -0,0 +1,659 @@
+    'En iyi CTF deneyimi içi lütfen pencerenizin boyutlarını artırın.',
+  'Thank you.' =>
+    'Teşekkürler.',
+  'Logout' =>
+    'Çıkış Yap',
+  'Registration' =>
+    'Kayıt',
+  'Play CTF' =>
+    'CTF Oyna',
+  'Rules' =>
+    'Kurallar',
+  //Translations for GameboardController
+  'Admin' =>
+    'Admin',
+  'ADMIN' =>
+    'ADMIN',
+  'Navigation' =>
+    'Naigasyon',
+  'View Mode' =>
+    'Görüntüleme Modu',
+  'View mode' =>
+    'Görüntüleme modu',
+  'Tutorial' =>
+    'Tutorial',
+  'Scoreboard' =>
+    'Skor Tablosu',
+  'You' =>
+    'Sen',
+  'Others' =>
+    'Diğerleri',
+  'All' =>
+    'Herkes',
+  'Leaderboard' =>
+    'Liderlik Tablosu',
+  'Announcements' =>
+    'Duyurular',
+  'Teams' =>
+    'Takımlar',
+  'Filter' =>
+    'Filtre',
+  'Activity' =>
+    'Aktivite',
+  'Game Clock' =>
+    'Oyun Saati',
+  //Translations for AdminController
+  'Auto' =>
+    'Otomatik',
+  'All Categories' =>
+    'Tüm Kategoriler',
+  'Open' =>
+    'Açık',
+  'Tokenized' =>
+    'Parçalanmış',
+  'Hour' =>
+    'Saat',
+  'Hours' =>
+    'Saat',
+  'Used by' =>
+    'Kullanan',
+  'Used By' =>
+    'Kullanan',
+  'Available' =>
+    'Müsait',
+  'Registration Tokens' =>
+    'Kayıt token\'ları',
+  'Create More' =>
+    'Daha Fazla Yarat',
+  'Export Available' =>
+    'Dışarı aktarmak mümkün',
+  'Not started yet' =>
+    'Henüz başlamadı',
+  'Configuration' =>
+    'Ayarlar',
+  'Tokens' =>
+    'Token\'lar',
+  'Game Configuration' =>
+    'Oyun Ayarları',
+  'OK' =>
+    'Tamam',
+  'status_' =>
+    'durum_',
+  'On' =>
+    'Açık',
+  'Off' =>
+    'Kapalı',
+  'Player Names' =>
+    'Oyuncu Adları',
+  'Players Per Team' =>
+    'Takım Başına Oyuncu',
+  'Registration Type' =>
+    'Kayıt Türü',
+  'Strong Passwords' =>
+    'Güçlü Şifreler',
+  'Team Selection' =>
+    'Takım Seçimi',
+  'Game' =>
+    'Oyun',
+  'Scoring' =>
+    'Puanlama',
+  'Progressive Cycle (s)' =>
+    'İlerleyici Döngü(ler)',
+  'Refresh Gameboard' =>
+    'Oyun Panosunu Yenile',
+  'Default Bonus' =>
+    'Varsayılan Bonus',
+  'Bases Cycle (s)' =>
+    'Temel Döngü(ler)',
+  'Default Bonus Dec' =>
+    'Varsayılan Bonus Dec',
+  'Timer' =>
+    'Süre Ölçer',
+  'Server Time' =>
+    'Sunucu Saati',
+  'Game Duration' =>
+    'Oyun Süresi',
+  'Begin Time' =>
+    'Başlangıç Saati',
+  'Expected End Time' =>
+    'Tahmini Bitiş Saati',
+  'Language' =>
+    'Dil',
+  'DELETE' =>
+    'SİL',
+  'Delete' =>
+    'Sil',
+  'No Announcements' =>
+    'Duyuru yok',
+  'Game Controls' =>
+    'Oyun Kontrolleri',
+  'Write New Announcement here' =>
+    'Buraya Yeni Duyuru yazın',
+  'Create' =>
+    'Yarat',
+  'General' =>
+    'Genel',
+  'Back Up Database' =>
+    'Veritabanı Yedekle',
+  'Export Full Game' =>
+    'Tüm Oyunu Dışa Aktar',
+  'Import Full Game' =>
+    'Tüm Oyunu Dışarıdan Al',
+  'Import Teams' =>
+    'Takımları Dışarıdan Al',
+  'Export Teams' =>
+    'Takımları Dışa Aktar',
+  'Import Logos' =>
+    'Logoları Dışarıdan Al',
```

Comment:
Seems like the .docx-formatted dump you sent me suggests the change be made on line 261, as well, instead of line 259.